source/fundamentals/crud/read-operations/project.txt

```diff
@@ -0,0 +1,148 @@
+==============================
+Specify Which Fields to Return
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In this guide, you will learn how to control which fields appear in
+documents returned by read operations.
+
+Many read requests require only a subset of fields in a document.
+For example, when logging a user in you may only need their username, and
+not all of their profile information. By default, queries in MongoDB return
+all fields in matching documents. You can use a **projection** to return
+only the data you need.
+
+A projection is a document that instructs MongoDB which fields of a
+document to return. Use the :ref:`Projections <projections-builders>` class
+to construct a projection document.
+
+Behavior
+--------
+
+Projections work in two ways:
+
+- Explicitly including fields. This has the side-effect of implicitly
+  excluding all unspecified fields.
+
+- Implicitly excluding fields. This has the side-effect of implicitly
+  including all unspecified fields.
+
+These two methods of projection are mutually exclusive: if you
+explicitly include fields, you cannot explicitly exclude fields, and
+vice versa.
+
+.. important::
+
+   The ``_id`` field is **not** subject to these mechanics. You must
+   explicitly exlude the ``_id`` field if you do not want it returned.
+   You can exclude the ``_id`` field even if you have specified certain
+   fields to include.
+
+Explanation
+-----------
+
+Consider the following collection containing documents that describe
+varieties of fruit:
+
+.. code-block:: json
+
+     { "_id": 1, "name": "apples", "qty": 5, "rating": 3 },
+     { "_id": 2, "name": "bananas", "qty": 7, "rating": 1 },
+     { "_id": 3, "name": "oranges", "qty": 6, "rating": 2 },
+     { "_id": 4, "name": "avocados", "qty": 3, "rating": 5 },
+
+In the following query, pass the projection to return the ``name``
+field of each document:
+
+.. code-block:: java
+   :emphasize-lines: 3
+
+   // return all documents with only* the name field
```

Comment:
Did you mean `*only*`?

Reply:
I think either formatting works. I've changed it to *only*